### PR TITLE
[CSFS-156] Build a To-Do App with REACT

### PR DESCRIPTION
**Pull request by Tocyn**
Screenshot 2021-05-09 at 22 35 37
Screenshot 2021-05-09 at 22 34 56

### CODE REVIEW

File: submissions/oluwatosinOke/react/src/App.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+          </div>
+        </form>
+      </div>
+    </div>
+  );
+}
+
+function App() {
+  const [todosRemaining,setTodosRemaining] = useState(0); 
+  const [todos, setTodos] = React.useState([
```
Comment (ToluAkin) [inline submissions/oluwatosinOke/react/src/App.js:51]: @Tocyn You are not expect to have the list in state. You are meant to create the list in the browser and push the values in to the array here.

**Review by ToluAkin — CHANGES_REQUESTED** [review]
Well done @Tocyn

**Comment by Tocyn** [comment]
@ToluAkin Kindly Review

**Comment by Tocyn** [comment]
@ToluAkin Kindly Review

**Comment by Tocyn** [comment]
@ToluAkin 
Changes have been effected.
Kindly Review

File: submissions/oluwatosinOke/react/src/App.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
+import React, {useState, useEffect} from "react";
+import "./App.css";
+import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (ToluAkin) [inline submissions/oluwatosinOke/react/src/App.js:3]: You are  not meant to use any css library for this project. Kindly remove and all instances of the its implementation in classes and js @Tocyn

File: submissions/oluwatosinOke/react/src/index.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+
+ReactDOM.render(
+  <React.StrictMode>
```
Comment (ToluAkin) [inline submissions/oluwatosinOke/react/src/index.js:8]: Remove this `<React.StrictMode>`

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Comment by Tocyn** [comment]
@ToluAkin 
Requested changes have been effected.
Kindly Review.

**Comment by Tocyn** [comment]
@ToluAkin 
I have effected the requested changes.
Kindly review

File: submissions/oluwatosinOke/react/src/index.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
```
Comment (ToluAkin) [inline submissions/oluwatosinOke/react/src/index.js:5]: REmove this in package.json and here

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/oluwatosinOke/react/src/index.js (hunk 4 above)
Comment (ToluAkin) [reply submissions/oluwatosinOke/react/src/index.js:5]: Make sure to run `npm install` again after updating the package.json

**Comment by ToluAkin** [comment]
@Tocyn I noticed that after unchecking an item, the color change is still there and the cross line. Also, your page is not mobile responsive which should not be.

**Comment by Tocyn** [comment]
@ToluAkin 
Requested changes have been effected
Kindly review.
Screenshot 2021-05-14 at 14 11 31

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@Tocyn I mentioned that when I unclick a list on the todo, it is still crossed out and color unchanged. Also your work is not mobile responsive.

**Comment by ToluAkin** [comment]
Click on this [link](image) to see the responsiveness I am having on my end

**Comment by Tocyn** [comment]
@ToluAkin 
The image you sent is not what i have.
what do I do?

**Comment by ToluAkin** [comment]
> @ToluAkin 
> The image you sent is not what i have.
> what do I do?

Could attach images in mobile and tablet screen from your end here? Also, as the checked and unchecked issue been fixed?

**Comment by Tocyn** [comment]
@ToluAkin 
Requested changes have been effected
Kindly Review
Screenshot 2021-05-14 at 19 41 43
Screenshot 2021-05-14 at 19 40 17

**Comment by Tocyn** [comment]
@ToluAkin
Kindly review

**Comment by Tocyn** [comment]
@ToluAkin 
kindly review

**Review by ToluAkin — APPROVED** [review]
Well done @Tocyn. Good Job

**Pull request merged**